adding scheduler cli-examples (#8983)
aws/aws-cli: awscli/examples/scheduler/untag-resource.rst

**To remove one or more tags from the specified EventBridge Scheduler schedule group**

The following ``untag-resource`` example removes one or more tags from the specified EventBridge Scheduler schedule group. ::

    aws scheduler untag-resource \
        --resource-arn "arn:aws:scheduler:ca-central-1:123456789012:schedule-group/DailyLambda" \
        --tag-keys "environment"

This command produces no output.

> For more information, see `This is the topic title <https://docs.aws.amazon.com/eventbridge/laDailyLambda/userguide/eb-pipes.html>`__ in the *Amazon EventBridge User Guide*.

Reviewer: 404 Not Found

Author: updated

**To remove one or more tags from the specified EventBridge Scheduler schedule group**

> The following ``untag-resource`` removes one or more tags from the specified EventBridge Scheduler schedule group. If the command succeeds, no output is returned. ::

Reviewer 2: ```suggestion
The following ``untag-resource`` example removes one or more tags from the specified EventBridge Scheduler schedule group. ::
```

Author: file fixed

	aws scheduler untag-resource \
		--resource-arn "arn:aws:scheduler:ca-central-1:123456789012:schedule-group/DailyLambda" \

> 		--tag-keys "environment"

Reviewer 2: ```suggestion
		--tag-keys "environment"

This command produces no output.
```
